src/main/java/com/collegecompendium/backend/controllers/StudentController.java

```diff
@@ -44,6 +46,19 @@ public Student getStudentById(@PathVariable String id, HttpServletResponse respo
 		return query.get();
 	}
 	
+	@GetMapping("/student")
+	public Student getCallingStudent(@AuthenticationPrincipal Jwt token, HttpServletResponse response) {
+		Student result = studentRepository.findDistinctByAuth0Id(token.getSubject());
```

Comment:
`findDistinctByAuth0Id` does not exist on the StudentRepository.

Reply:
I have no idea why that didn't flag on my desktop?